mwdb/model/migrations/versions/3648d869ef45_.py

```diff
@@ -0,0 +1,27 @@
+"""empty message
+
+Revision ID: 3648d869ef45
+Revises: 4f0405411201
+Create Date: 2021-07-20 10:03:58.015298
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "3648d869ef45"
+down_revision = "4f0405411201"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("api_key", sa.Column("name", sa.String(length=100), nullable=False))
```

Comment:
I guess this should be somehow extended if we're adding a non-nullable column in the database with some existing API keys - their names should be set to empty strings?

Reply:
Fixed that on my own (including other backend things): https://github.com/CERT-Polska/mwdb-core/pull/420/commits/d7f5ae7897b6ff0af4ed7ad6baa8f95f72881c3b

Reply:
Thanks :+1: